esmvaltool/cmorizers/obs/cmorize_obs_merra2.py

```diff
@@ -0,0 +1,157 @@
+"""ESMValTool CMORizer for MERRA2 data.
+
+Tier
+    Tier 3: restricted datasets (i.e., dataset which requires a registration
+ to be retrieved or provided upon request to the respective contact or PI).
+
+Source
+    https://goldsmr4.gesdisc.eosdis.nasa.gov/data/MERRA2_MONTHLY/
+
+Last access
+    20191129
+
+Download and processing instructions
+    - For download instructions see the download script `download_merra2.sh`.
+
+"""
+import glob
+import logging
+import os
+from copy import deepcopy
+from datetime import datetime
+
+import cf_units
+import iris
+from dask import array as da
+
+from esmvalcore.cmor.table import CMOR_TABLES
+
+from . import utilities as utils
+
+logger = logging.getLogger(__name__)
+
+
+def _fix_time_monthly(cube):
+    """ This function fixes the time coordinate by setting it
+        to the 15th of each month """
+    # Read dataset time unit and calendar from file
+    dataset_time_unit = str(cube.coord('time').units)
+    dataset_time_calender = cube.coord('time').units.calendar
+    # Convert datetime
+    time_as_datetime = cf_units.num2date(cube.coord('time').core_points(),
+                                         dataset_time_unit,
+                                         dataset_time_calender)
+    newtime = []
+    for timepoint in time_as_datetime:
+        midpoint = datetime(timepoint.year, timepoint.month, 15)
+        newtime.append(midpoint)
+
+    newtime = cf_units.date2num(newtime,
+                                dataset_time_unit,
+                                dataset_time_calender)
+    # Put them on the file
+    cube.coord('time').points = newtime
+    cube.coord('time').bounds = None
+    return cube
+
+
+def _load_cube(in_files, var):
+    cube_list = iris.load_raw(in_files)
+    selected = [c for c in cube_list if c.var_name == var['raw']]
+    selected = iris.cube.CubeList(selected)
+
+    drop_attrs = ['History', 'Filename', 'Comment', 'RangeBeginningDate',
+                  'RangeEndingDate', 'GranuleID', 'ProductionDateTime',
+                  'Source']
+    drop_time_attrs = ['begin_date', 'begin_time',
+                       'time_increment', 'valid_range', 'vmax', 'vmin']
+    for cube in selected:
+        for attr in drop_attrs:
+            cube.attributes.pop(attr)
+        for attr in drop_time_attrs:
+            cube.coord('time').attributes.pop(attr)
+        cube.coord('time').points = cube.coord(
+            'time').core_points().astype('float64')
+
+    from iris.util import unify_time_units
```

Comment:
could u put this in the main list of imports pls and then run an `isort` on it so it is all nicely sorted :beer: